ISISComputingGroup/ibex_gui
# Ticket1404 lakeshore336 opi

###Description of work

Added OPI for Lakeshore 336 Temperature Controller
The OPI has multiple tabs consistently with the corresponding LabView screen: one for each of the two Outputs (where the user can actually control a channel), one listing all the four Inputs (just for temperature readbacks), and two containing temperature graphs (one with all Input T and setpoints, one only with Input T)

Note that for each Output the user can specify which input it should control. When switching input, the Input box of the Ouput panels should update correspondingly.

###To test

This is for Ticket ISISComputingGroup/IBEX#1404
See ticket for list of all related pull requests

###Acceptance criteria

* The user can start/stop the IOC from the GUI
* The user can see the IPADDR macro under Configurations
* The user can add a Lakeshore336 in a synoptic
* The user can control the device from the OPI
* The OPI is easy enough to use and understand!

---

#### Code Review


### Functional Tests
